config/locales/en.yml

```
@@ -1978,6 +1979,7 @@ en:
   label_wiki_start: "Start page"
   label_work_package: "Work package"
   label_work_package_attachments: "Work package attachments"
+  label_work_package_file_link_plural: "Work package file links"
```

```suggestion
  label_work_package_file_link_plural: "Links between work packages and files in external file storages"
```
@fraber Here also, "file link" is not a easy to understand concept for normal users/admins. That is why we almost no where use that term, apart from the permissions.

CC @apfohl

---

Again very difficult to find a good name. So here is my argumentation, please override later if that is important to you.
As with "Project storages" above, "Links between work packags and files in external file storages" does not match the style of other options on that page like "Work package attachments" or "Work package categories".

Also, the list of options could be ordered alphabetically. maybe create a Ticket for it?